tasks/deploy.js

```diff
@@ -0,0 +1,194 @@
+    res = await deployTestNetworkDao(deployFunction, network, hardhatImports);  
+  } else if (network === "ropsten") {
+    res = await deployTestNetworkDao(deployFunction, network, hardhatImports);
+  } else if (network === "test" || network === "coverage") {
+    const accounts = await hre.ethers.getSigners();
+    res = await deployTestDao(deployFunction, network, accounts,hardhatImports);
+  }
+
+  let { dao, extensions, adapters, testContracts, votingHelpers, factories  } = res;
+  if (dao) {
+    await dao.finalizeDao();
+    const finalizeAdapters = formattedOject(adapters, "Adapter");
+    const finalizeExtensions = formattedOject(extensions, "Extension");
+    const finalizeTestContracts = formattedOject(testContracts, "Contract");
+    const finalizeVotingHelpers = formattedOject(votingHelpers, "Contract");
+    const finalizeFactories = formattedOject(factories, "Contract");
+
+    console.log("************************ Adapters ************************");
+    console.table(finalizeAdapters);
+
+    console.log("************************ Extensions ************************");
+    console.table(finalizeExtensions);
+
+    console.log("************************ TestContracts ************************");
+    console.table(finalizeTestContracts);
+
+    console.log("************************ VotingHelpers ************************");
+    console.table(finalizeVotingHelpers);
+
+    console.log("************************ Factories ************************");
+    console.table(finalizeFactories);
+
+    console.log("************************");
+    console.log(`DaoRegistry: ${dao.address}`);
+    console.log(`BankExtension: ${extensions.bank.address}`);
+    console.log(
+      `NFTExtension: ${extensions.nft ? extensions.nft.address : ""}`
+    );
+    console.log(
+      `ERC20Extension: ${
+        extensions.erc20Ext ? extensions.erc20Ext.address : ""
+      }`
+    );
+    console.log("************************");
+  } else {
+    console.log("************************");
+    console.log("no migration for network " + network);
+    console.log("************************");
+  }
+});
+
+const capitalize = str => str.charAt(0).toUpperCase() + str.slice(1);
+
+const formattedOject = (object, name) => {
+  const array = [];
+  for (let key in object) {
+    array.push({
+      name: capitalize(key) + name,
+      address: object[key] ? object[key].address : ""
+    });
+  }
+  return array;
+}
+
+async function deployTestDao(deployFunction, network, accounts, hardhatImports) {
+  if (!process.env.DAO_NAME)
+    throw Error("Missing env var: DAO_NAME");
+  if (!process.env.ERC20_TOKEN_NAME)
+    throw Error("Missing env var: ERC20_TOKEN_NAME");
+  if (!process.env.ERC20_TOKEN_SYMBOL)
+    throw Error("Missing env var: ERC20_TOKEN_SYMBOL");
+  if (!process.env.ERC20_TOKEN_DECIMALS)
+    throw Error("Missing env var: ERC20_TOKEN_DECIMALS");
+
+  return await deployDao({
+    ...hardhatImports,
+    deployFunction,
+    unitPrice: unitPrice,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819147)
```suggestion
    unitPrice,
```